Source/WebKit/UIProcess/WebProcessCache.cpp

```diff
@@ -154,30 +154,53 @@ bool WebProcessCache::addProcess(Ref<CachedProcess>&& cachedProcess)
     if (!canCacheProcess(process))
         return false;
 
+#if PLATFORM(COCOA) || PLATFORM(GTK) || PLATFORM(WPE)
+        cachedProcess->startSuspensionTimer();
+#endif
+
+    if (process->isSharedProcess()) {
+        auto site = process->sharedProcessSite();
+        ASSERT(site);
+        if (auto previousProcess = m_sharedProcessesPerSite.take(*site))
+            WEBPROCESSCACHE_RELEASE_LOG("addProcess: Evicting shared process from WebProcess cache because a new process was added for the same domain", previousProcess->process().processID());
+
+        evictAtRandomIfNeeded();
+
+        WEBPROCESSCACHE_RELEASE_LOG("addProcess: Added shared process to WebProcess cache (size=%u, capacity=%u) %" PRIVATE_LOG_STRING, cachedProcess->process().processID(), size() + 1, capacity(), site->toString().utf8().data());
```

Comment:
Since it's a RELEASE_LOG this probably needs to be SENSITIVE_LOG_STRING.

Reply:
It's not though, right? It only logs the size & capacity of the cache.